package/MDAnalysis/analysis/encore/clustering/ClusterCollection.py

```diff
@@ -120,7 +120,7 @@ def __len__(self):
 
     def add_metadata(self, name, data):
         if len(data) != self.size:
-            raise TypeError("Size of metadata is not equal to the number of"
+            raise TypeError("Size of metadata is not equal to the number of "
                             "cluster elmements")
```

Comment:
This is a typo, should be "elements" please do change 


```suggestion
                            "cluster elements")
```

Reply:
I fixed it. Sorry for not noticing it at the first place